moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -863,6 +863,20 @@ std::vector<OMPLPlannerDescription> MoveItConfigData::getOMPLPlanners() const
   spar_stwo.addParameter("max_failures", "5000", "maximum consecutive failure limit. default: 5000");
   planner_des.push_back(spar_stwo);
 
+  OMPLPlannerDescription aitstar("AITstar", "geometric");
+  aitstar.addParameter(
+      "use_k_nearest", "1",
+      "Set whether to use a k-nearest RGG connection model. If false, AITstar uses an r-disc model. Default: 1");
```

Comment:
```suggestion
      "whether to use a k-nearest RGG connection model (1) or an r-disc model (0). Default: 1");
```
other descriptions seem to skip the verb